setup.py

```diff
@@ -11,7 +11,7 @@
         "pytest>=8.0,<9.0",
         "pytest-cov>=4.1,<5.0",
         "pytest-instafail>=0.4,<1.0",
-        "pytest-xdist>=3.5,<4.0",
+        "pytest-xdist>=3.0,<3.4",
```

Comment:
Is there an issue that requires downgrading like this?

Reply:
yes, it's in one of the commit messages

Reply:
https://github.com/pytest-dev/pytest-xdist/issues/1034